examples/cifar_demo/README.md

```diff
@@ -0,0 +1,28 @@
+# Demo
+
+This demo walks the user through setting up Clipper, deploying models,
+and querying Clipper for predictions.
+
+## Setup
+
+Before you can run the demo, you must have access to a machine running
+Docker that you can SSH into. If running on EC2, you can use AMI `ami-3ba0f05b`,
+an Ubuntu image that has Docker and Docker-Compose installed.
```

Comment:
If we can get https://github.com/ucbrise/clipper/pull/60 merged, we should include information about running the demo locally. This seems like a very critical addition, as there is arguably much less overhead to installing Docker locally than launching AWS instances.